models/Task.php

```diff
@@ -1055,23 +1055,36 @@ public function getColor()
 
     // ###########  handle task related account  ###########
 
-    public function AddTaskAccount()
+    public function manageTaskAccount()
```

Comment:
is there actually some kind of afterCreate callback? so we don't have to bloat up that method that much?

Reply:
what is the EVENT_AFTER_INSERT event?

Reply:
I think we should call the method on every *SCENARIO_EDIT* , if there is already an account created we just set the title and the responsible and if not we create an account ? 

re :  EVENT_AFTER_INSERT  , it's an event dispatched by an *ACTIVE RECORD* whenever there is an insertion into database, but in our case we should check even on *SCENARIO_EDIT* maybe the responsible or the task name which will be displayed in the space accounts list have been changed.

Reply:
ok, but we could split the method into two smaller ones. one for the updating and one for the creation. 
but I guess like this it is easier to create an account for already existing tasks?